src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -43,7 +43,7 @@ setup_teuthology() {
 
     ${TEUTHOLOGY_PYTHON_BIN:-/usr/bin/python3} -m venv venv
     source venv/bin/activate
-    pip install -U pip 'setuptools >= 12'
+    pip install -U pip 'setuptools>=12,<60'
```

Comment:
Good catch @djgalloway ! Thanks a lot... I was an old fan of rolling deps, but I'm becoming more and more conservative when it comes to leaving things run loose 🙈